tree/tree/src/TLeafD32.cxx

```diff
@@ -205,8 +208,11 @@ void TLeafD32::Streamer(TBuffer &R__b)
    if (R__b.IsReading()) {
       R__b.ReadClassBuffer(TLeafD32::Class(), this);
 
-      if (fTitle.Contains("["))
-	 fElement = new TStreamerElement(Form("%s_Element", fName.Data()), fTitle.Data(), 0, 0, "Double32_t");
+      if (fTitle.Contains("[")) {
```

Comment:
What is the title is 'just' `[n]`?

Reply:
I think this was right before, with the strchee, type is d or d[0,0] whereas title is arr[n] so we only need to check for type and not fTitle?

Reply:
I expect the content of fTitle to be one of
```
[n]
[n]/[0,10,10]
[0,10,10]
(empty)
```
(and I don't recall how the fixed size array is encoded)

Reply:
In master, without these changes, when calling Branch, I get (just before overwriting fTitle):
```
fTitle = arr[n]
type = "d[0,0,10]"
```

Reply:
With arr_default, I get:
fTitle=arr2[n]
type = d